GOVERNANCE-subprojects.md

```diff
@@ -0,0 +1,176 @@
+# [TODO:PROJECTNAME] Project Governance
```

Comment:
We should add a disclaimer saying something like `This is a suggested template and not required by the CNCF` to avoid any confusion (don't want us to be accused of "forcing a governance model").

Reply:
Might make more sense to stick such a disclaimer on https://github.com/cncf/project-template/blob/main/README.md rather then each template.

Reply:
Yeah, probably.  I mean, the fact that there are multiple templates available would seem to indicate that, but one can't be too careful.